client/components/register/imports/RegisterForm.jsx

```diff
@@ -348,14 +350,23 @@ class RegisterForm extends Component {
 
   _holdHarmless() {
     if (!this.state.showHoldHarmless) return <p></p>;
-    return (
-      <Segment basic>
-        <p>I hereby acknowledge that I have voluntarily chosen (or voluntarily chosen to allow my minor child) to participate in the {eventYear} WWU Great Puzzle Hunt sponsored by the WWU Mathematics Department, held on {eventDay}, {eventDate}. I understand the risks involved in the Puzzle Hunt, including the unlikely but potential risk of injury to me (or my minor child), and I agree to accept any and all risks associated with my participation.</p>
+    {/*<Segment basic>
```

Comment:
Why is this commented out and then re-added below..? If we don't need this old hold harmless, can we just delete it?

Reply:
It just seems like we often revert to older versions of text from when there was an in-person only event, I'm happy to delete it if we think it won't be used though.